json-schema/schema.json

```diff
@@ -1,140 +1,338 @@
+          ],
+          "properties": {
+            "type": {
+              "const": "Feature"
+            },
+            "properties": {
+              "allOf": [
+                {
+                  "required": [
+                    "ssys:targets"
+                  ]
+                },
+                {
+                  "$ref": "#/definitions/ssys"
+                }
+              ]
+            },
+            "assets": {
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/definitions/ssys"
+              }
+            }
+          }
+        },
+        {
+          "$ref": "#/definitions/stac_extensions"
+        }
+      ]
+    },
+    {
+      "$comment": "This is the schema for STAC Collections.",
+      "allOf": [
+        {
+          "type": "object",
+          "required": [
+            "type",
+            "ssys:targets"
+          ],
+          "properties": {
+            "type": {
+              "const": "Collection"
+            },
+            "assets": {
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/definitions/ssys"
+              }
+            },
+            "item_assets": {
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/definitions/ssys"
+              }
+            }
+          }
+        },
+        {
+          "$ref": "#/definitions/stac_extensions"
+        },
+        {
+          "$ref": "#/definitions/ssys"
+        }
+      ]
+    },
+    {
+      "$comment": "This is the schema for STAC Catalogs.",
+      "allOf": [
+        {
+          "type": "object",
+          "required": [
+            "type",
+            "ssys:targets"
+          ],
+          "properties": {
+            "type": {
+              "const": "Catalog"
+            },
+            "assets": {
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/definitions/ssys"
+              }
+            },
+            "item_assets": {
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/definitions/ssys"
+              }
+            }
+          }
+        },
+        {
+          "$ref": "#/definitions/stac_extensions"
+        },
+        {
+          "$ref": "#/definitions/ssys"
+        }
+      ]
+    }
+  ],
+  "definitions": {
+    "stac_extensions": {
+      "type": "object",
+      "required": [
+        "stac_extensions"
+      ],
+      "properties": {
+        "stac_extensions": {
+          "type": "array",
+          "contains": {
+            "const": "https://raw.githubusercontent.com/thareUSGS/ssys/main/json-schema/schema.json"
+          }
+        }
+      }
+    },
+    "ssys": {
+      "type": "object",
+      "properties": {
+        "ssys:targets": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "oneOf": [
```

Comment:
I do not believe that this needs to make use of oneOf. I think just:

```json
"type":"array",
"minItems": 1,
"items": {
  "type":"string",
  "enum": [
    "Mercury",
    "Venus",
    "..."
    ]
},
"additionalItems": false
```

That should force at least one item from the enum be in the targets list.